### PR TITLE
OOP School Library : Basic UI 

### PR DESCRIPTION
**Pull request by HtetWaiYan7191**
### This pull request includes the following changes : 

- Create a app.rb file for required methods 
- Create a main.rb  file to show the basic UI 
- Fix linter errors 

Thanks for reviewing my codes 😁

### CODE REVIEW

File: class/app.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  def self.create_book
+    print 'Title: '
+    title = gets.chomp.to_s
+    print 'Author: '
+    author = gets.chomp.to_s
+    Book.new(title, author)
+    puts 'Book created successfully'
+  end
+
+  def self.create_rental
```
Comment (Davidosky007) [inline class/app.rb:68]: * Kindly fix this error i get in my console when i try to create rentals
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @HtetWaiYan7191 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Interactive console app built 👍 
* Well written ruby codes 👍 
* No linter errors 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: class/app.rb (hunk 1 above)
Comment (HtetWaiYan7191) [reply class/app.rb:68]: Hello @Davidosky007  sir, It is working on my pc  sir. Can you please explain me how can I solve these . Thank you very much for your time 😁 
Screenshot (154)
Screenshot (155)
Screenshot (156)

Comment (Davidosky007) [reply class/app.rb:68]: For some wired reason it didn't work on my end , kindly resubmit for review and sorry for the inconvenience thanks

Comment (HtetWaiYan7191) [reply class/app.rb:68]: Thanks sir. I will resubmit again

**Review by Davidosky007 — APPROVED** [review]
Hi @HtetWaiYan7191 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well written PR 👍 
* Good looking readme 👍 
* No linter errors 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**